ciphey/basemods/Decoders/baudot.py

```diff
@@ -0,0 +1,114 @@
+import re
+from typing import Optional, Dict, List
+
+from ciphey.iface import ParamSpec, Config, T, U, Decoder, registry
+
+@registry.register_multi((str, str), (bytes, bytes))
+class Baudot(Decoder[str, str]):
+    def decode(self, ctext: T) -> Optional[U]:
+        """Write the code that decodes here
+        ctext -> the input to the function
+        returns string
+    """
+        baudot_map = {
```

Comment:
This should go into CipheyDists! I'll need to write documentation for this :)

Reply:
You can always figure it out yourself :+1: https://github.com/Ciphey/CipheyDists/tree/master/cipheydists/translate

Convert your dictionaries into json files (with appropriate names) and add them to this folder in cipheydists.

And then we can easily call your dictionaries (see the other decoders for how to)

Reply:
i made a pull request for cipheydists can you check the changes i made?